thermal.snbt

```diff
@@ -0,0 +1,1201 @@
+			}]
+			title: "No Itemducts?"
+			x: -3.0d
+			y: 3.0d
+		}
+		{
+			dependencies: ["01E9F0D9D11372C2"]
+			description: ["Turbo-Servos work over 20 times faster than normal Servos. However, it has less filtering functionality."]
+			id: "083BED9C03893B87"
+			subtitle: "Craft &eTurbo-Servos&r"
+			tasks: [{
+				count: 2L
+				id: "62076C314405E631"
+				item: { Count: 5, id: "thermal:turbo_servo_attachment" }
+				type: "item"
+			}]
+			x: -4.0d
+			y: 7.0d
+		}
+		{
+			dependencies: ["623878F31D5F2EF2"]
+			description: ["Fluid cells can store large amounts of liquids. Fluid cells can be upgraded to store even more with &eIntegral Components!&r"]
+			id: "56FDF04F9F56BF8A"
+			subtitle: "Craft a &eFluid Cell&r"
+			tasks: [{
+				id: "1FD2CDE9FE1D31D9"
+				item: {
+					Count: 1
+					id: "thermal:fluid_cell"
+					tag: {
+						BlockEntityTag: {
+							TankInv: [{
+								Amount: 0
+								Capacity: 32000
+								FluidName: "minecraft:empty"
+								Tank: 0b
+							}]
+						}
+					}
+				}
+				type: "item"
+			}]
+			x: -4.0d
+			y: 5.0d
+		}
+		{
+			dependencies: ["7BC63DBDAC493B1C"]
+			description: ["Redstone Flux cells can store large amounts of power. Redstone Flux cells can be upgraded to store even more with &eIntegral Components!&r"]
+			id: "1ADA4A86A0600792"
+			subtitle: "Craft a &eRedstone Flux Cell&r"
+			tasks: [{
+				id: "0CE1D51B2D21DE1F"
+				item: {
+					Count: 1
+					id: "thermal:energy_cell"
+					tag: {
+						BlockEntityTag: {
+							Energy: 0
+							EnergyMax: 1000000
+							EnergyRecv: 1000
+							EnergySend: 1000
+						}
+					}
+				}
+				type: "item"
+			}]
+			x: -2.0d
+			y: 5.0d
+		}
+		{
+			dependencies: ["01E9F0D9D11372C2"]
+			description: ["Filters can be used to allow different fluids to go in different directions in a fluiduct system. "]
+			id: "0B6C86CEA29FC671"
+			subtitle: "Craft &eFilters&r"
+			tasks: [{
+				count: 2L
+				id: "71868710870814DE"
+				item: "thermal:filter_attachment"
+				type: "item"
+			}]
+			x: -4.0d
+			y: 8.0d
+		}
+		{
+			dependencies: ["7BC63DBDAC493B1C"]
+			description: ["Redstone Flux Limiters can be attached to fluxducts to limit the transfer of energy!"]
+			id: "7DD56A4D1C92A96E"
+			subtitle: "Craft &eRedstone Flux Limiters&r"
+			tasks: [{
+				count: 2L
+				id: "3FD4A12D0015A1FA"
+				item: { Count: 2, id: "thermal:energy_limiter_attachment" }
+				type: "item"
+			}]
+			x: -2.0d
+			y: 6.0d
+		}
+		{
+			dependencies: ["24CB532D44E6DC43"]
+			description: ["The Tinker's Workbench allows you to add upgrades to your &eThermal Series equipment&r to make them better!"]
+			id: "20442358C1AFD7B8"
+			subtitle: "Craft a &eTinker's Workbench&r"
+			tasks: [{
+				id: "6B1C0DA39F926BB0"
+				item: "thermal:tinker_bench"
+				type: "item"
+			}]
+			x: -5.0d
+			y: 0.0d
+		}
+		{
+			dependencies: ["1D9E37BBD5F54394"]
+			hide_dependency_lines: false
+			id: "10F5F913B5673B09"
+			subtitle: "Make some important alloys!"
+			tasks: [

```

Comment:
that's a lot of alloys. i'd personally say it might be requiring a bit too much of the more complex ones, so maybe we crank down the counts for those by atleast half.